landing-pages/site/static/.htaccess

```diff
@@ -3,7 +3,7 @@ RedirectMatch Permanent ^/docs/(stable|1.10.10)/(cli|macros)(\.html)?$ "https://
 RedirectMatch Permanent ^/((_api|_images|_modules|_sources|_static|howto)/.*)$ "https://airflow.apache.org/docs/apache-airflow/stable/$1"
 RedirectMatch Permanent ^/((1.10.1|1.10.2|1.10.3|1.10.4|1.10.5|1.10.6|1.10.7|1.10.8|1.10.9|1.10.10)/.*)$ "https://airflow.apache.org/docs/apache-airflow/$1"
 RedirectMatch Permanent ^/((api|changelog|cli|concepts|errors|faq|genindex|http-routingtable|installation|integration|kubernetes|license|lineage|macros|metrics|plugins|privacy_notice|profiling|project|py-modindex|scheduler|search|security|start|timezone|tutorial|ui)(\.html)?)$ "https://airflow.apache.org/docs/apache-airflow/stable/$1"
-RedirectMatch Permanent ^/docs/((1.10.1|1.10.2|1.10.3|1.10.4|1.10.5|1.10.6|1.10.7|1.10.8|1.10.9|1.10.10|1.10.11|1.10.12|1.10.13|stable)/.*)$ "https://airflow.apache.org/docs/apache-airflow/$1"
+RedirectMatch Permanent ^/docs/((1.10.1|1.10.2|1.10.3|1.10.4|1.10.5|1.10.6|1.10.7|1.10.8|1.10.9|1.10.10|1.10.11|1.10.12|1.10.13|1.10.14|2\.[0-9.]+|stable)/.*)$ "https://airflow.apache.org/docs/apache-airflow/$1"
```

Comment:
I don't think we need redirects for all versions as we didn't publish these links before.

The only problem is `https://airflow.apache.org/docs/2.0.0/` which was included in the latest release by mistake. The following solution seems to me to address our problem better.
```
Redirect Permanent /docs/2.0.0/ https://airflow.apache.org/docs/apache-airflow/2.0.0/